JeffShepherd/Overlook
# Feature/positioning and style

## What does this PR do?

- Adds positioning and style to app. Also adds media queries for improved responsiveness.

## How should it be tested in the UI?

- Run 'npm start' and ensure the page looks good in desktop, tablet, and mobile sizes.
